### PR TITLE
Adding security policy

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Supported Versions
+
+The latest release of Rancher Desktop is the supported version. If you have found a security vulnerability in an older release of Rancher Desktop please verify it is in the latest release.
+
+## Reporting a Vulnerability
+
+SUSE supports responsible disclosure and endeavors to resolve security issues in a reasonable timeframe. To report a security vulnerability, email security-rancher [at] suse.com. You may (but are not required to) use the PGP key `rsa4096/C9DF50BDAC351DA9` for encrypted communication. This [key can be found on keys.openpgp.org](https://keys.openpgp.org/search?q=security-rancher@suse.com).
+
+### GPG Key - `security-rancher@suse.com`
```
Comment (macedogm) [inline SECURITY.md:11]: s/GPG/PGP/

**Review by abonillabeeche — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (mattfarina) [reply SECURITY.md]: Good catch. Thanks.

**Review by rak-phillip — CHANGES_REQUESTED** [review]
Just one thing looks a little off to me

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Security Policy
+
+## Supported Versions
+
+The latest release of Rancher Desktop is the supported version. If you have found a security vulnerability in an older release of Rancher Desktop please verify it is in the latest release.
+
+## Reporting a Vulnerability
+
+SUSE supports responsible disclosure and endeavors to resolve security issues in a reasonable timeframe. To report a security vulnerability, email security-rancher [at] suse.com. You may (but are not required to) use the PGP key `rsa4096/C9DF50BDAC351DA9` for encrypted communication. This [key can be found on keys.openpgp.org](https://keys.openpgp.org/search?q=security-rancher@suse.com).
```
[the remark this answers is not in the document]
Comment (mattfarina) [reply SECURITY.md:9]: No. This was a request from the security team to limit bots parsing and spamming the list. As someone on security lists... spam can be annoying. I see every email coming into the lists I'm on to make sure spam filtering doesn't accidentally filter a real notification.

**Review by rak-phillip — APPROVED** [review]

**Review by rak-phillip — APPROVED** [review]

**Pull request merged**